app/views/historic_appointments/index.html.erb

```diff
@@ -0,0 +1,91 @@
+<% content_for :title, @content_item["title"] %>
+<% page_class "historic-appointments-index govuk-width-container" %>
+
+<%= render "govuk_publishing_components/components/breadcrumbs", {
+  collapse_on_mobile: true,
+  breadcrumbs: [
+    {
+      title: "Home",
+      url: "/",
+    },
+    {
+      title: "How government works",
+      url: "/government/how-government-works",
+    },
+    {
+      title: "History of the UK government",
+      url: "/government/history",
+    },
+  ]
+} %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <%= render "govuk_publishing_components/components/title", {
+      context: "History",
+      title: @content_item["title"],
+    } %>
+  </div>
+</div>
+
+<%# 21st century section %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
```

Comment:
We can use shared template that we introduced for past foreign secs / chancellors here